scylla-server/tests/systems-services.test.ts

```diff
@@ -1,16 +1,56 @@
-import prisma from '../src/prisma/prisma-client';
 import { describe, test, expect } from 'vitest';
+import { getAllSystems, upsertSystems } from '../src/services/systems.services';
 
-/**
- * Tests for CRUD service functions
- */
+import prisma from '../src/prisma/prisma-client';
 
 /**
- * basic ahh unit test for get all systems
+ * Tests for CRUD Service functions
  */
 describe('CRUD Systems', () => {
+  /**
+   * updated unit test for get all systems
+   */
   test('Get All Systems Works', async () => {
-    const allSystems = await prisma.system.findMany();
-    expect(allSystems).toStrictEqual([]);
+    const expected = [];
```

Comment:
Hmm maybe we can alter this test now to be like after an upsert one so that we can test that retrieving works